fragments/k8s/cdk/ci-info.yaml

```diff
@@ -0,0 +1,21 @@
+# This is an incomplete ci-info.yaml fragment.
+bundle:
+  namespace: kos.tsakalozos
```

Comment:
Is this left over from your personal work or do you mean to use your namespace here?